tests/foreman/cli/test_hammer.py

```
@@ -27,10 +31,7 @@ def _fetch_command_info(self, command):
                         found = True
                         break
             if not found:
-                self.fail(
-                    'Was not possible to fetch information for "{0}" command.'
-                    .format(command)
-                )
+                return None
```

Let the test fail only when all information is available.